src/airflow_docker/conf.py

```diff
@@ -9,8 +9,15 @@ def get_boolean_default(key, default):
         return default
 
 
-def get_default(key, default):
+def get_default(key, default=None):
     try:
         return conf.get("airflowdocker", key)
     except AirflowConfigException:
         return default
+
+
+def get_default_list(key, default=None):
+    result = get_default(key)
+    if result is None:
+        return []
```

Comment:
`return [] if default is None else default`

Reply:
i dont prefer this in general